x/fundraising/keeper/msg_server_test.go

```diff
@@ -164,6 +164,7 @@ func (s *KeeperTestSuite) TestMsgPlaceBid() {
 			types.NewMsgPlaceBid(
 				auction.GetId(),
 				bidder.String(),
+				types.BidTypeBatchWorth,
```

Comment:
For the test, `BidTypeBatchMany` can be also added to test in other parts.

Reply:
Yes, you can maybe add the cases when you handle https://github.com/tendermint/fundraising/issues/88 issue after this PR is merged? #88 is only about changing msg types so we can add a bunch of other keeper test cases in a separate issue.